roseWong/assignments/cyclicSort/lc268/lc268.py

```diff
@@ -0,0 +1,60 @@
+# Problem Statement #
+
+# We are given an array containing ‘n’ distinct numbers taken from the range 0 to ‘n’. Since the array has only ‘n’ numbers out of the total ‘n+1’ numbers, find the missing number.
+# given an array of length n and told that there is a missing number with 0 to n, we
+# need to sort the array, using the cyclic sort pattern, to find out the missing numbe
+# 
+
```

Comment:
Missing your approach/psuedocode and edge cases. These will be really helpful to let you interviewer know how your code achieves the prompt.